Feat mypage test

qbobl5 opened the pull request:
controller 작성 중에 오류 생겨서 그 파일빼고 올립니다.
제 코드 확인해주시고 시간 되시면 제 오류 좀 함께 해결해주실 수 있나요..?
코드 통일성 주려고 재성님 코드 참고해서 짜봤는데 테스트 해보니까 의존성 주입이 안 된다라는 오류가 뜨더라구요..
@Autowired 붙여보기도 했는데 안 돼서 여쭤봅니다ㅠ..

qbobl5 committed:
fix: param 오탈자 수정

이 부분 오탈자 같아서 수정해서 일단 pr 날려놓슴당~~ 확인해주세여

Co-Authored-By: Seohyeon Paek <137274073+shpaek@users.noreply.github.com>

qbobl5 committed:
Revert "fix: param 오탈자 수정"

This reverts commit c20e16491ff58416943c510c871347b64f394088.

qbobl5 committed:
feat: ResultDTO 추가

qbobl5 committed:
fix: ResultView 주석 위치 수정

qbobl5 committed:
feat: ResultService 추가

qbobl5 committed:
feat: ResultRepository 추가

qbobl5 committed:
fix: ResultDTO builder 어노테이션 추가

qbobl5 committed:
fix: ResultService 오타 수정

qbobl5 committed:
fix: ResultService 메소드 매개변수 추가

qbobl5 committed:
feat: findOrderByResultDateDesc 추가

qbobl5 committed:
feat: findResultListByMemberId 추가

회원이 푼 테스트 목록을 페이징 처리하여 조회하는 서비스 기능 구현

qbobl5 committed:
fix: query 메소드명 오류 수정

qbobl5 committed:
fix: 메소드명 오탈자 수정

leejaeseong11 commented on src/main/java/com/timekiller/zzatool/result/service/ResultServiceImpl.java:
```diff
@@ -0,0 +1,58 @@
+
+@Service
+@RequiredArgsConstructor
+public class ResultServiceImpl implements ResultService {
```
```java
@Service
public class ResultServiceImpl implements ResultService {
    private static final Logger logger = Logger.getLogger(TestServiceImpl.class.getName());
    @Autowired private ResultRepository resultRepository;
    @Autowired private TestRepository testRepository;
```
@qbobl5 이렇게하면 잘 되는 것 같은데 물어보신게 이게 맞는지 모르겠네요

qbobl5 committed:
fix: service 오류 수정

qbobl5 committed:
feat: controller-myResultList 추가

qbobl5 committed:
fix: ResultRepository-리턴 타입 변경

qbobl5 committed:
fix: ResultServiceImpl 수정

1. findResultListByMemberId: repo 메소드 리턴 타입 변경하면서 코드 변경
2. findTestTitle: null 예외 발생 시 처리할 try~catch 추가

qbobl5 committed:
feat: myResultList-회원이 푼 테스트 목록 조회 호출하는 controller 메소드 추가
